src/app/frontend/deploy/deployfromfile.html

```diff
@@ -0,0 +1,35 @@
+<!--
+Copyright 2015 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<div layout="row" layout-align="space-between start">
+  <div layout="column" flex="80">
+    <md-input-container class="md-block">
+      <label>YAML or JSON file</label>
+      <input ng-model="ctrl.fileName" ng-disabled="true"/>
+    </md-input-container>
+  </div>
+  <div layout="column" flex="18">
```

Comment:
Flex=20 is too much for the upload button. Then the upload button is has more width then the other buttons in the card like deploy and cancel. If I want to complete the total to 100 by using
Flex=82, Flex:18 then upload file input field is shrinked. It is kind of a fine adjustment. I don't want to assign an exact width to the button as well to make it compatible with the other buttons.